sql/src/main/java/io/crate/analyze/OrderBy.java

```diff
@@ -79,7 +79,7 @@ public OrderBy subset(Collection<Integer> positions) {
         boolean[] reverseFlags = new boolean[positions.size()];
         int pos = 0;
         for (Integer i : positions) {
-            orderBySymbols.add(this.orderBySymbols.get(i));
+            orderBySymbols.add(Symbols.DEEP_COPY.apply(this.orderBySymbols.get(i)));
```

Comment:
Otherwise the symbols are mutated somewhere (I think due to normalization) and then the InputColumns cannot be created anymore because of a field/reference miss-match.